includes/classes/Feature/DidYouMean/DidYouMean.php

```diff
@@ -0,0 +1,395 @@
+<?php
+/**
+ * Did You Mean feature.
+ *
+ * @since   4.6.0
+ * @package elasticpress
+ */
+
+namespace ElasticPress\Feature\DidYouMean;
+
+use ElasticPress\{Elasticsearch, Feature, FeatureRequirementsStatus, Features };
+
+/**
+ * Did You Mean feature class.
+ */
+class DidYouMean extends Feature {
+
+	/**
+	 * Initialize feature, setting it's config.
+	 */
+	public function __construct() {
+		$this->slug = 'did-you-mean';
+
+		$this->title = esc_html__( 'Did You Mean', 'elasticpress' );
+
+		$this->summary = __( '"Did You Mean" search feature provides alternative suggestions for misspelled or ambiguous search queries, enhancing search accuracy and user experience.', 'elasticpress' );
```

Comment:
```suggestion
		$this->summary = __( 'Recommend alternative search terms for misspelled queries or terms with no results', 'elasticpress' );
```

per https://github.com/10up/ElasticPress/issues/2089#issuecomment-1579014160

Reply:
I missed that comment. updated the summary.